src/components/pages/ProductionSchedule.vue

```diff
@@ -580,7 +590,7 @@ export default {
       daysOffByPerson: {},
       draggedEntities: [],
       endDate: moment().add(6, 'months').endOf('day'),
-      isSidePanelOpen: false,
+      entityTypeFilter: 'ALL',
```

Comment:
`isSidePanelOpen` must be declare on `data()`

```suggestion
      entityTypeFilter: 'ALL',
      isSidePanelOpen: false,
```